src/__tests__/unit/lib/e-mem/index.test.ts

```diff
@@ -0,0 +1,193 @@
+import {EMemModel} from '../../../../lib';
+//import { ERRORS } from '../../../../util/errors';
+//const { InputValidationError } = ERRORS;
+
+jest.setTimeout(30000);
+
+describe('lib/e-mem: ', () => {
+  describe('init AzureImporterModel: ', () => {
+    it('initalizes object with properties.', async () => {
+      const eMemModel = await new EMemModel();
+      expect(eMemModel).toHaveProperty('authenticate');
+      expect(eMemModel).toHaveProperty('configure');
+      expect(eMemModel).toHaveProperty('execute');
+    });
+  });
+
+  describe('configure(): ', () => {
+    it('configures model instance with given params.', async () => {
+      const params = {};
+      const model = await new EMemModel().configure(params);
+      expect(model).toBeInstanceOf(EMemModel);
+    });
+  });
+
+  describe('authenticate(): ', () => {
```

Comment:
if model doesn't have authenticate method, you can just drop it instead of checking if it's undefined